charts/coturn/values.yaml

```diff
@@ -108,3 +108,7 @@ livenessProbe:
 readinessProbe:
   timeoutSeconds: 5
   failureThreshold: 5
+
+# If you need to specify which IP Coturn should bind to.
```

Comment:
Could you move this whole block up to line 25, so that it is next to listening port?

Reply:
@julialongtin moved as requested. requesting review so it can be merged.